### PR TITLE
Invite link

### PR DESCRIPTION
**Pull request by sei-noconnor**

### CODE REVIEW

**Review by sei-tspencer — APPROVED** [review]

**Comment by sei-ebram** [comment]
You mentioned about squashing the commits but you don't have to, that'll automatically happen when we merge.
Also, I noticed that you used 2-space indents but our standard convention is 4

**Comment by sei-aschlackman** [comment]
> You mentioned about squashing the commits but you don't have to, that'll automatically happen when we merge.
> Also, I noticed that you used 2-space indents but our standard convention is 4

Yes, please change the indents to 4 spaces. It's hard to parse the diffs with all the indentation changes.

File: Alloy.Api/Hubs/EventHub.cs [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  public class EventHub : Hub
+  {
+    private readonly AlloyContext _context;
+
+    public EventHub(AlloyContext context)
+    {
+      _context = context;
+    }
+
+    public async Task JoinEvent(Guid eventId)
```
Comment (sei-aschlackman) [inline Alloy.Api/Hubs/EventHub.cs]: All these Join methods should have some sort of permissions check.

**Review by sei-aschlackman — CHANGES_REQUESTED** [review]

**Review by sei-aschlackman — APPROVED** [review]
